### ISSUE TITLE
There seems no implementation of positional_encoding

### ISSUE THREAD

**Issue by zhouruirui1998** [Atze00/MoViNet-pytorch#32]
Hello, thanks for the PyTorch implementation!
There seems no implementation of positional_encoding,does that influence in the stream model?

**Comment by zhouruirui1998** [comment]
sorry,I just realized A0-A3 don't need positional_encoding

**Comment by gabrielegoletto** [comment]
Hi @zhouruirui1998, could you please explain why you mentioned that they are not needed with A0-A3?

> sorry,I just realized A0-A3 don't need positional_encoding

**Comment by mkmohangb** [comment]
@ezius07 From [here](https://github.com/tensorflow/models/tree/master/official/projects/movinet#streaming-models): 

Note: A3, A4, and A5 models use a positional encoding in the squeeze-excitation blocks, while A0, A1, and A2 do not. For the smaller models, accuracy is unaffected without positional encoding, while for the larger models accuracy is significantly worse without positional encoding.

**Comment by gabrielegoletto** [comment]
@usernameandme 
I had not seen it! Many thanks!
